Resources/Prototypes/Entities/Clothing/Neck/medals.yml

```diff
@@ -19,4 +19,15 @@
   - type: Sprite
     sprite: Clothing/Neck/Medals/gold.rsi
   - type: Clothing
-    sprite: Clothing/Neck/Medals/gold.rsi  
\ No newline at end of file
+    sprite: Clothing/Neck/Medals/gold.rsi
+
+- type: entity
+  parent: ClothingNeckBase
+  id: ClothingNeckEngineerMedal
+  name: Enginer medal

```

Comment:
enginerd1!!

use `name: engineer medal` instead

Reply:
Already changed the name